mvnw

```diff
@@ -21,288 +21,219 @@
 # ----------------------------------------------------------------------------
 # Apache Maven Wrapper startup batch script, version 3.2.0
 #
-# Required ENV vars:
```

Comment:
Why this has changes? It's unrelated, please undo

Reply:
Can you extract this to a separate commit?

Reply:
We tend to split changes into smaller, atomic commits (and we test that each commit separately compiles) and keep commit messages in sync with the actual commit content.